source/renderer/app/components/widgets/Dialog.js

```diff
@@ -37,7 +37,7 @@ export type DialogActionOptions = {
 export type DialogActions = DialogActionItems | DialogActionOptions;
 
 type Props = {
-  title?: string,
+  title?: string | Element<any>,
```

Comment:
@daniloprates just a tip. You can use type `Node` from React types instead of `Element<any>`

Reply:
Done ✓